.github/workflows/greet.yml

```
@@ -22,26 +22,26 @@ jobs:
         uses: actions/first-interaction@v3
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          issue_message: |
+          issue-message: |
```

actionlint (1.7.11)

[error] input "issue-message" is not defined in action "actions/first-interaction@v3". available inputs are "issue_message", "pr_message", "repo_token"

(action)